src-docs/src/views/xy_chart_bar/bar_example.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
```

Comment:
include another example showing a 2-series non-stacked bar chart to demonstrate the clustering

Reply:
If you check the second example on Bar charts, the one called "Stacked Vertical Bar Chart", just click on the button toggle stacked, and you can see the example of 2 series non stacked bar chart.